tensorflow/core/grappler/optimizers/auto_mixed_precision.h

```diff
@@ -24,6 +24,12 @@ namespace grappler {
 
 enum class AutoMixedPrecisionMode { CUDA, MKL };
 
+// Note: This is primarily used by the tf.experimental.tensorrt. Converter class
+// to use mixed precision on ops not converted by TensorRT. It is also used for
+// the soon-to-be-deprecated enable_mixed_precision_graph_rewrite API.
+// Do not remove this API or comment without contacting:
```

Comment:
Would you please modify the last sentence to something like this:
If you will need to modify or remove this API, please be kindly contact the TF-TRT owners.

Reply:
We really can't leave our names ? I'm afraid that they would only contact the Google TF-TRT owner in this case. I would also like to be informed in case someone plan to make any change to this API.

Reply:
reedwm@ believes that we should completely remove this comment, and make sure we will have TF-TRT tests failing if the pass is removed.